src/app/search/search.module.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by Ilya on 04/11/15.
+ */
+
+'use strict';
+
+import SearchDirective from './search.directive';
+import SearchController from './search.controller';
+import './search.resource';
+
+angular.module('gh.search', ['gh.api.search'])
+    .controller('SearchController', SearchController)
```

Comment:
> Try to eliminate this line.

As I understand you mean controller. I can declare it in directive but I don't want to. Because there might be case when I need to test directive only without instantiating controller.